File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.java
class Solution {
    public int solution(String my_string, String is_prefix) {
        String[] answer = new String[my_string.length()];
        for(int i = 0; i < my_string.length(); i++){
            answer[i] = my_string.substring(0, i);
        }
        
        for(int i = 0; i < answer.length; i++){
            if(is_prefix.equals(answer[i])) return 1;
        }
        return 0;
    }
}

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.js
function solution(my_string, is_prefix) {
    let answer = [];
    for(let i = 0; i < my_string.length; i++){
        answer.push(my_string.slice(0, i))
    }
    for(const str of answer){
        if(str == is_prefix) return 1
    }
    return 0;
}